webpack.config.js

```diff
@@ -4,7 +4,7 @@ var webpack = require('webpack')
 var HtmlWebpackPlugin = require('html-webpack-plugin')
 
 if (process.env.NODE_ENV === 'production') {
-    var publicPath = 'https://neontribe.github.io/ARCVMarket/';
```

Comment:
While you can dev locally with ./ it needs to be the full https to run if pushed to github pages (at the moment - needs fixing); avoid pushing webpack.config.js OR put it back before you do :-)